BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -100,11 +100,7 @@ public class PortfolioStore: ObservableObject {
             return
           }
           let symbol = token.symbol.lowercased()
-          if let oldBalance = balances[token.symbol] {
-            balances[symbol] = oldBalance + balance
-          } else {
-            balances[symbol] = balance
-          }
+          balances[symbol, default: 0] += balance
```

Comment:
oh i love the default value for dictionary in swift

Reply:
Yeah I agree. Had I used it first time I wouldn't have introduced this bug 🙃